README.md

```diff
@@ -125,7 +150,22 @@ Current state | Next word | Next state
 
 ## A2.4
 
-«replace this with your answer»
+Current state | Next word | Next state
+--------------|-----------|-----------
+    S0        |    the    |     S1
+    S0        |    the    |     S2
+    S1        |    lazy   |     S1
+    S1        |    lazy   |     S2
+    S1        |   smelly  |     S1
+    S1        |   smelly  |     S2
+    S2        |    dog    |     S3
+    S2        |    cat    |     S3
+    S3        |    ate    |     S4
+    S3        |    ran    |     S4
+    S4        |  slowly   |     S5
+    S4        |  noisily  |     S5
+    S4        |    EOI    |    END
+    S5        |    EOI    |    END
```

Comment:
This table accurately reflects the diagram, except it gets confused about the double arrows between states 1 and 2. It's incorrect in the big scheme of things, but I'm grading it based on your diagram.

# -2